docs/examples/Transformer Examples.ipynb

```diff
@@ -1585,7 +1588,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.12.2"
+   "version": "3.10.14"
```

Comment:
out of scope of this ticket, but it's probably a good idea to bump the python version you use to a more recent one

Reply:
Then its' probably good idea to bump the version on the build guide as well

Reply:
good point. out of scope here